insta/insta.html

```diff
@@ -16,8 +16,12 @@
   <script type="text/javascript" src="insta.js"></script>
 </head>
 <body onload="init()">
-  <header><h1><img id="humburger" src='../images/humburger_button.png'></img>Taxeedee: <span id="subtitle">Travels of Olga and Jack</span></h1></header>
-  <main>
+  <header><h1><img id="humburger" src='../images/humburger_button.png'></img> Taxeedee: <span id="subtitle">Travels of Olga and Jack</span></h1></header>
+    <main>
+      <span id="dropdown" style="display:none">
+      About Us</br>
```

Comment:
Should the first option be "Home" or "Posts"?

Reply:
👍